homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -397,8 +397,14 @@ async def async_step_setup_serial_manual_path(self, user_input=None):
 
     async def async_step_import(self, import_config=None):
         """Handle the initial step."""
-        await self.async_set_unique_id(DOMAIN)
-        self._abort_if_unique_id_configured()
+        entry = await self.async_set_unique_id(DOMAIN)
+        # In version 0.113 devices is not stored in entry.data, to be safe update data for all VERSION=1 entries
+        if entry:
+            if entry.version == 1:
+                entry.version = 2
+                self._abort_if_unique_id_configured(import_config)
+            else:
+                self._abort_if_unique_id_configured()
         return self.async_create_entry(title="RFXTRX", data=import_config)
```

Comment:
Just noticed that we don't validate the connection to the device on import. Normally we do that also on import.

Reply:
It was on my radar to do so. Shall I do it in this PR or make a separate one? I don't think it's a lot of code, the logic is already there.

Reply:
If it's not a lot, either way is fine. You decide.

Reply:
I'll make a separate PR, all tests except config flow tests fail because they use `async_setup_component`. I think it would be better to refactor to `MockConfigEntry` but it's more than anticipated.